plan/plans.go

```diff
@@ -108,6 +108,14 @@ type Simple struct {
 	Statement ast.StmtNode
 }
 
+// InsertGeneratedColumns is for completing generated columns in Insert.
+// We resolve generation expressions in plan, and eval those in executor.
+type InsertGeneratedColumns struct {
+	Columns []*ast.ColumnName
```

Comment:
because there is `ast.ColumnName` in `Insert`. And, the offset of a column to schema is calculated in `executor`, not in `ResolveIndices`. I prefer keep it as original in this PR. If doing that in `plan` is better than `executor`, I can do that in next PR.